lightning-invoice/src/payment.rs

```diff
@@ -1766,9 +1742,15 @@ mod tests {
 		assert_eq!(inflight_map.len(), 5);
 	}
 
-	struct TestRouter;
+	struct TestRouter {
+		scorer: RefCell<TestScorer>,
+	}
 
 	impl TestRouter {
+		fn new(scorer: RefCell<TestScorer>) -> Self {
```

Comment:
Let's hide the `RefCell` part of the interface by having this method wrap the scorer.

Reply:
Fixed in 9f34cbfe